src/main/java/com/dev/cinema/model/dto/UserRegistrationDto.java

```diff
@@ -2,8 +2,8 @@
 
 import com.dev.cinema.validation.annotation.EmailValidatorConstraint;
 import com.dev.cinema.validation.annotation.FieldsValueMatch;
-import jakarta.validation.constraints.Min;
-import jakarta.validation.constraints.NotNull;
+import javax.validation.constraints.Min;
```

Comment:
can this annotation be applied to String?

Reply:
value of the field or property must be an integer value greater than or equal to the number in the value element - so I`ll remove it and hope won't do this again in a future. =)

Reply:
you may use `@Size` for String)